extensions/amp-story/0.1/test/test-amp-story.js

```diff
@@ -221,15 +305,6 @@ describes.realWin('amp-story', {
     expect(hideBookendStub).to.have.been.calledOnce;
   });
 
-  // TODO(newmuis/amphtml-story#187): Re-enable this test.
-  it.skip('should return a valid page count', () => {
```

Comment:
Any particular reason to remove this?

Reply:
I moved it to the first test. Should have kept the right order to help the review